vendor-extra/ContentPageBundle/src/DependencyInjection/ContentPageExtension.php

```diff
@@ -26,9 +26,9 @@ private function addPage(string $name, array $config, ContainerBuilder $containe
     {
         $id = sprintf(self::CONTENT_CONTROLLER_ID, $name);
         $definition = new Definition(ContentController::class);
-
+       
+        $definition->setAutowired(true);
```

Comment:
Autowiring's discourages for bundles. We'll need to make the client configurable (will most likely be one content page per content store, which has one client).